### PR TITLE
Update depreciation.py

### PR DESCRIPTION
**Pull request by gautam1858**
writing if statement in one line

### CODE REVIEW

File: tensorflow/python/util/deprecation.py [hunk 1]
```diff
@@ -161,18 +161,16 @@ def __init__(self, *args, **kwargs):
...
-                'From %s: The name %s is deprecated. Please use %s instead.\n',
-                _call_location(), deprecated_name, name)
-        super(NewClass, self).__init__(*args, **kwargs)
-
+          _PRINTED_WARNING[NewClass.__init__] = True
+          logging.warning(
+              'From %s: The name %s is deprecated. Please use %s instead.\n',
+              _call_location(), deprecated_name, name)
+          super(NewClass, self).__init__(*args, **kwargs)
+
```
Comment (martinwicke) [inline tensorflow/python/util/deprecation.py]: This is a change in behavior: 

- super needs to be outside the if
- logging.warning is outside the if warn_once

You can pull the check for _PRINTED_WARNING out, but not the warn_once.

**Review by martinwicke — CHANGES_REQUESTED** [review]
it

**Comment by gautam1858** [comment]
@martinwicke made the changes

File: tensorflow/python/util/deprecation.py [hunk 2]
```diff
@@ -161,18 +161,16 @@ def __init__(self, *args, **kwargs):
...
-            logging.warning(
-                'From %s: The name %s is deprecated. Please use %s instead.\n',
-                _call_location(), deprecated_name, name)
+          _PRINTED_WARNING[NewClass.__init__] = True
+        logging.warning(
+            'From %s: The name %s is deprecated. Please use %s instead.\n',
+            _call_location(), deprecated_name, name)
         super(NewClass, self).__init__(*args, **kwargs)
-
+
```
Comment (martinwicke) [inline tensorflow/python/util/deprecation.py]: This is still not right. Please make sure that the behavior matches the original. Right now, you would always log a warning, not only the first time.

If you're unsure, run the tests. I think they at least cover the "warn once" case, not sure the coverage for _PRINT_DEPRECATION_WARNINGS is good enough, but passing the tests will be a decent indicator.

**Review by martinwicke — CHANGES_REQUESTED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Assignee @martinwicke: It has been 17 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by tensorflowbutler** [comment]
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.

**Pull request closed without merging**